lndr-backend/src/Lndr/Db/Identity.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Lndr.Db.Identity where
+
+import           Control.Monad
+import           Data.Maybe (listToMaybe)
+import           Data.Text (Text)
+import qualified Data.Text as T
+import           Database.PostgreSQL.Simple
+import           Lndr.Db.Types
+import           Lndr.Types
+import           Network.Ethereum.Web3
+import           Text.EmailAddress
+import qualified Text.EmailAddress as Email
+
+addVerificationStatus :: VerificationStatusEntry -> Connection -> IO Int
+addVerificationStatus statusValues@(VerificationStatusEntry address id status) conn = fromIntegral <$>
+    execute conn "INSERT INTO identity_verification (address, applicant_id, status) VALUES (?,?,?) ON CONFLICT (address) DO UPDATE SET status = EXCLUDED.status" (address, id, status)
```

Comment:
I had to look this one up, since it's the first time I've really looked at an *up-sert* in PostgreSQL.
For posterity: `EXCLUDED` comes from PostgreSQL's notion of the newly inserted row and exists only when there is a conflict to resolve. This SQL statement instructs to insert a new record, and if the address conflicts with an existing record, just update the existing record's status to the new status. :+1:

Reply:
I had to look it up too haha